bin/zsnap

```diff
@@ -16,6 +16,7 @@ from zfstools.util import stderr, verbose_stderr, set_verbose
 parser = optparse.OptionParser("usage: %prog [-k NUMSNAPS] <datasetname>")
 parser.add_option('-k', '--keep', action='store', dest='keep', default=7, help='how many snapshots to keep (default: %default), 0 means delete all')
 parser.add_option('-p', '--prefix', action='store', dest='prefix', default="autosnapshot-", help='prefix to prepend to snapshot names (default: %default)')
+parser.add_option('-P', '--property', action='store', dest='property', help='property (name=value) to apply to snapshots')
```

Comment:
This should also document that the filtering method changes when this option is specified.  Otherwise there'll be angry users with bug reports who will go "why did this thing not delete that snapshot?"